giovaninb/LacrasteCloud
# Fetch images from CK

Description:
This merge adds code to fetch and show images on custom tableView cell

How to test:
1. Open the App;
2. Click on Guest;
3. Check that the screen allows images on cells;

Evidence:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/17914434/179817756-93d68d73-6c20-4fa5-8c74-b31066161a18.png">
